courses/machine_learning/deepdive2/image_classification/solutions/2_mnist_models.ipynb

```diff
@@ -19,7 +19,7 @@
   },
```

Comment:
<p>Is there a reason calling test.py was deleted?</p><p>!python3 -m mnist_trainer.trainer.test</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/705/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>Can't get it to work. May need to ping Danielle</p>